### DISCUSSION TITLE
第3节课作业(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#179] [作业]
这里是12班第3节课作业提交位置~

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135502100

**Comment by Jeff-Huang-SHU** [comment]
https://blog.csdn.net/2301_77209094/article/details/135482787?spm=1001.2014.3001.5502

**Comment by ltstar** [comment]
[课程3.md](https://github.com/InternLM/tutorial/files/13896975/3.md)

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec3/

**Comment by canxingliuying** [comment]
https://blog.csdn.net/weixin_42449385/article/details/135565757

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/1a1dc7130573423eaccd3c58c566e2b0

**Comment by Smiling-Weeping-zhr** [comment]
https://www.cnblogs.com/smiling-weeping-zhr/p/17963782

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135584805?spm=1001.2014.3001.5502

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/GoJQdniJdoyj9rxxeW7cVyt1ndf

**Comment by JiangXiaobai00** [comment]
https://d0elb2210mg.feishu.cn/docx/Gj6gdtMgqoCwIixQ1Wrce0BDnce?from=from_copylink

**Comment by siuserjy** [comment]
https://blog.csdn.net/weixin_46334272/article/details/135660960

**Comment by ShenFeilang** [comment]
https://shenfeilang.notion.site/InternLM-f59131c72d574a04b5b3b59fc6654017?pvs=4

**Comment by 0-yy-0** [comment]
https://blog.csdn.net/qq_34354951/article/details/135832697

**Comment by sunhanyu714** [comment]
https://github.com/sunhanyu714/Internlm_learning/blob/main/%E7%AC%AC%E4%B8%89%E7%AB%A0%E4%BD%9C%E4%B8%9A.md
